docker_compose/README.md

```diff
@@ -0,0 +1,9 @@
+# Docker compose for sycamore
+
+Runs the entire sycamore stack + aryn demo UI.
+
+See compose.yaml for details and extensions, but in short:
+
+1. cd .../docker_compose
```

Comment:
Three dots?  Is that a linux thing?

Reply:
It was to indicate whatever is your path to the docker_compose directory.
Changed to cd path/to/sycamore/docker_compose